problem_builder/public/css/mentoring.css

```diff
@@ -62,6 +63,10 @@
     display: table; /* Enable line-wrapping in IE8 */
 }
 
+.mentoring legend p:last-child {
+    margin-bottom: 0 !important;
```

Comment:
Sure, I can rewrite this to use a longer (more specific) selector, so that it can still override the existing style in the LMS that's causing the issue here.